testing/regress/ecl/setup/soaptest_getdocumentcount.ecl

```diff
@@ -0,0 +1,7 @@
+//nothor
```

Comment:
I think even in the regression suite we usually put a comment block at the top of every ecl file.